publish.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+GIT_REPO=https://github.com/tbouron/clocker
```

Comment:
This probably isn't the correct repo here?

Reply:
Ooops, you are right, that was for my test. Updated @drigodwin